config.yaml

```diff
@@ -1,5 +1,5 @@
 # The title is the title of your Add-On
-title: Hello World
+title: Summarize
```

Comment:
I know this is still a work in progress, but for completeness sake, you should update the description here, and also the properties - I believe you do not need any properties, unless you want to add some options.

Reply:
Done.